src/app/pages/my-account/my-account.component.scss

```diff
@@ -21,12 +21,42 @@
 			border-radius: 50%;
 			margin: 0 auto;
 			position: relative;
-			button {
+			img {
+				height: 112px;
+				width: 112px;
+				object-fit: cover;
+				border-radius: 50%;
+			}
+			&--picker {
 				position: absolute;
-				right: 0px;
-				bottom: 0px;
-				background-color: white;
-				color: $color-primary-400;
+				right: 0;
+				bottom: 0;
+				border-radius: 50%;
+				background-color: $color-white;
+				display: flex;
+				justify-content: center;
+				justify-items: center;
+				align-items: center;
+				width: 40px;
+				height: 40px;
+				label {
+					display: flex;
+					flex-direction: center;
+					justify-content: center;
+					align-items: center;
```

Comment:
As a future task we can look into SASS mixins to not repeat so much many blocks of code